src/index.html

```diff
@@ -152,7 +153,65 @@ <h2 class="modal__title">New post</h2>
         </div>
       </div>
     </div>
-    <script defer src="./scripts/validation.js"></script>
-    <script defer src="./scripts/index.js"></script>
+
+    <div class="modal" id="delete-modal">
+      <div class="modal__content" id="delete-content">
+        <button type="button" class="modal__close-btn" id="close-btn"></button>
+        <h2 class="modal__title" id="delete-title">
+          Are you sure you want to delete this image?
+        </h2>
+        <form
+          name="remove-form"
+          class="modal__form"
+          id="delete-form"
+          novalidate
+        >
+          <button type="submit" class="modal__submit-btn" id="delete-btn">
+            Delete
+          </button>
+          <button type="button" class="modal__submit-btn" id="cancel-btn">
+            Cancel
+          </button>
+        </form>
+      </div>
+    </div>
+    <div class="modal" id="avatar-modal">
+      <div class="modal__content">
+        <button
+          type="button"
+          class="modal__close-btn"
+          id="avatar__close-btn"
+        ></button>
+        <h2 class="modal__title" id="edit-avatar-title">Edit profile avatar</h2>
+        <form
+          name="avatar-form"
+          class="modal__form"
+          id="edit-avatar-form"
+          novalidate
+        >
+          <label
+            for="profile-avatar-input"
+            class="modal__label modal__label_type_avatar"
+          >
+            Profile Picture
+            <input
```

Comment:
* Please, add `type="url"` for the `avatar` input, to validate it as a `link` https://snipboard.io/C6Y4pq.jpg